spec/fixtures/items.yml

```diff
@@ -2,3 +2,10 @@ one:
   name: Thingy
   description: a great Thingy
   unit_price: 123456
+  merchant_id: 980190962
+
```

Comment:
not sure if we'll have a problem with hard-coding ids; they might change as we run the specs...